WordPress/Tests/WhiteSpace/SemicolonSpacingUnitTest.inc

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/*
+ * Test that the sniff does *not* throw incorrect errors for semi-colons in
```

Comment:
Clarification:
"for semi-colons" => "for spaces before semi-colons"

Reply:
Fixed, just used slightly different phrasing (whitespace instead of spaces as whitespace can also be tabs or new lines)